frontend/components/Home/EmailForm.tsx

```diff
@@ -28,6 +28,7 @@ export const EmailForm: FC = () => {
 
   const submit = ({ email }: FormValues): void => {
     generateLink({ variables: { data: { email } } });
+    trackEvent({ eventName: 'generateLinkClicked' });
```

Comment:
This might get confusing after we introduce more and more event, and small typos might make the data inconsistent in the Plusible.
Can you please make an object (or enum) of these events in the `frontend/analytics/plausible.ts` file and use it that way?

Reply:
Thanks, updated :)